### DISCUSSION TITLE
How to clear VDH

### DISCUSSION THREAD

**Discussion by joonid** [aclap-dev/video-downloadhelper#297] [Help and Q&A]
### Your question

How can I reset/clear/restart VDH?  Sometimes the extension thinks it has successfully accomplished a download but it hasn't, or there is an accumulation of errors that I would like to clear, etc. I would rather not have to restart my browser. Is there a simple way to clear VDH?

I also recommend this topic for the VDH How To pages.

(VDH 8.2.1.1 on Firefox on Windows) 

### Addon & CoApp version and details

(VDH 8.2.1.1 on Firefox on Windows)

**Comment by GrampaWildWilly** [comment]
Go into the VDH settings and . . .

![image](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/0382f854-21f7-4835-920d-cec3866a742b)

Click the Reload extension entry in that menu.

**Comment by joonid** [comment]
@GrampaWildWilly   Nice - thank you.
